bats/sql-conflicts.bats

```diff
@@ -0,0 +1,190 @@
+  [ "$status" -eq 0 ]
+  [[ ! "$output" =~ "base" ]]
+  [[ "$output" =~ \+[[:space:]]+\|[[:space:]]+ours[[:space:]] ]] || false
+  [[ "$output" =~ \+[[:space:]]+\|[[:space:]]+theirs[[:space:]] ]] || false
+
+  dolt sql -q "DELETE from dolt_conflicts_one_pk WHERE our_pk1 = 0"
+  dolt sql -q "DELETE from dolt_conflicts_two_pk WHERE our_pk1 = 0 and our_pk2 = 0"
+
+  EXPECTED=$( echo -e "table,num_conflicts")
+  run dolt sql -r csv -q "SELECT * FROM dolt_conflicts"
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "$EXPECTED" ]] || false
+}
+
+
+@test "modify conflict" {
+  dolt SQL -q "INSERT INTO one_pk (pk1,c1,c2) VALUES (0,0,0)"
+  dolt SQL -q "INSERT INTO two_pk (pk1,pk2,c1,c2) VALUES (0,0,0,0)"
+  dolt add .
+  dolt commit -m "initial values"
+  dolt branch feature_branch master
+  dolt SQL -q "UPDATE one_pk SET c1=1,c2=1 WHERE pk1=0"
+  dolt SQL -q "UPDATE two_pk SET c1=1,c2=1 WHERE pk1=0 and PK2=0"
+  dolt add .
+  dolt commit -m "changed master"
+  dolt checkout feature_branch
+  dolt SQL -q "UPDATE one_pk SET c1=2,c2=2 WHERE pk1=0"
+  dolt SQL -q "UPDATE two_pk SET c1=2,c2=2 WHERE pk1=0 and PK2=0"
+  dolt add .
+  dolt commit -m "changed feature_branch"
+  dolt checkout master
+  dolt merge feature_branch
+
+  EXPECTED=$( echo -e "table,num_conflicts\none_pk,1\ntwo_pk,1")
+  run dolt sql -r csv -q "SELECT * FROM dolt_conflicts"
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "$EXPECTED" ]] || false
+
+  EXPECTED=$( echo -e "base_pk1,our_pk1,their_pk1,base_c1,our_c1,their_c1,base_c2,our_c2,their_c2\n0,0,0,0,1,2,0,1,2" )
+  run dolt sql -r csv -q "SELECT base_pk1,our_pk1,their_pk1,base_c1,our_c1,their_c1,base_c2,our_c2,their_c2 FROM dolt_conflicts_one_pk"
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "$EXPECTED" ]] || false
+
+  EXPECTED=$( echo -e "base_pk1,our_pk1,their_pk1,base_pk2,our_pk2,their_pk2,base_c1,our_c1,their_c1,base_c2,our_c2,their_c2\n0,0,0,0,0,0,0,1,2,0,1,2" )
+  run dolt sql -r csv -q "SELECT base_pk1,our_pk1,their_pk1,base_pk2,our_pk2,their_pk2,base_c1,our_c1,their_c1,base_c2,our_c2,their_c2 FROM dolt_conflicts_two_pk"
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "$EXPECTED" ]] || false
+
+  run dolt conflicts cat one_pk
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "base" ]]
+  [[ "$output" =~ \*[[:space:]]*\|[[:space:]]+ours[[:space:]] ]] || false
+  [[ "$output" =~ \*[[:space:]]*\|[[:space:]]+theirs[[:space:]] ]] || false
+
+  run dolt conflicts cat two_pk
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "base" ]]
+  [[ "$output" =~ \*[[:space:]]*\|[[:space:]]+ours[[:space:]] ]] || false
+  [[ "$output" =~ \*[[:space:]]*\|[[:space:]]+theirs[[:space:]] ]] || false
+
+  dolt sql -q "DELETE from dolt_conflicts_one_pk WHERE our_pk1 = 0"
+  dolt sql -q "DELETE from dolt_conflicts_two_pk WHERE our_pk1 = 0 and our_pk2 = 0"
+
+  EXPECTED=$( echo -e "table,num_conflicts")
+  run dolt sql -r csv -q "SELECT * FROM dolt_conflicts"
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "$EXPECTED" ]] || false
+}
+
+@test "delete modify conflict" {
```

Comment:
How does this test differ from the one above? Not obvious to me

Reply:
this one does deletes of the rows in the feature branch.  The other one updates in the different branch.